mezzanine/pages/templates/admin/pages/page/change_list.html

```diff
@@ -11,15 +11,15 @@
 </style>
 <script>window.__page_ordering_url = '{% url "admin_page_ordering" %}';</script>
 {% if not settings.USE_MODELTRANSLATION %}
-<script src="{% static "mezzanine/js/"|add:settings.JQUERY_UI_FILENAME %}"></script>
+<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
```

Comment:
Is there a reason we're eliminating this setting?

Reply:
As jQuery UI is now listed in grapelli-safe's base.html we don't need it here.